TLM/TLM/UI/ModUI.cs

```diff
@@ -87,6 +95,20 @@ public class UIOpacityObservable : GenericObservable<UIOpacityNotification> {
             // One time load
             TMPELifecycle.Instance.TranslationDatabase.ReloadTutorialTranslations();
             TMPELifecycle.Instance.TranslationDatabase.ReloadGuideTranslations();
+
+            StartCoroutine(StartTMPEToolCoroutine());
```

Comment:
make Awake return `IEnumerator` and just `yield return new StartTMPEToolCoroutine()` or.. merge both methods 😛

Reply:
Oh wow! I didn't know that is possible!

Reply:
@krzychu124 what I have done has the benefit of being executed only after ModUI is enabled. so I like to keep it this way.
The code is not going to be simpler because try ... catch block cannot contain yield return

Therefore I like to keep it this way.